fragdenstaat_de/settings/base.py

```diff
@@ -375,6 +374,7 @@ def three_days_ago_but_not_sundays(date):
     }
 
     FILER_ENABLE_PERMISSIONS = True
+    FILER_MIME_TYPE_WHITELIST = ["*"]
```

Comment:
Why is this needed? Are there any implications?

Reply:
django-filer 3.0 introduces file validation. Previously, all files were accepted without checks. That line keeps this behaviour. We could configure it more strictly, see [docs](https://django-filer.readthedocs.io/en/latest/validation.html#validation).

Reply:
But isnt this equivalent to the default? The docs say

> If FILER_MIME_TYPE_WHITELIST is empty, all mime types will be accepted (default behaviour).